S17-supply/zip.t

```diff
@@ -44,7 +44,24 @@ for ThreadPoolScheduler.new, CurrentThreadScheduler -> $*SCHEDULER {
 
     throws-like( { Supply.zip(42) },
       X::Supply::Combinator, combinator => 'zip' );
-
 }
 
+subtest {
+    my $*SCHEDULER = ThreadPoolScheduler.new;
+    react {
+        whenever Promise.in(1) {
```

Comment:
I'd bump this up to 5 seconds or so, just in case; systems can be under unusual load during spectest runs.